internal/identity/identitymanager.go

```diff
@@ -115,7 +119,7 @@ func (im *identityManager) NormalizeSigningKey(ctx context.Context, inputKey str
 	// If the caller is not confident that the blockchain plugin/connector should be used to resolve,
 	// for example it might be a different blockchain (Eth vs Fabric etc.), or it has its own
 	// verification/management of keys, it should set `assets.keyNormalization: "none"` in the config.
-	if keyNormalizationMode != KeyNormalizationBlockchainPlugin {
+	if keyNormalizationMode != KeyNormalizationBlockchainPlugin || im.blockchain == nil {
```

Comment:
I think removing this check would be enough to make that happen... since it would fall through to `normalizeKeyViaBlockchainPlugin` and error from there.